### ISSUE TITLE
Deselecting Bluetooth Device Deactivates it

### ISSUE THREAD

**Issue by hscissors** [twilio/audioswitch#36]
**Bug**
When deselecting bluetooth device, sometimes the device becomes inactive and is no longer listed in `availableAudioDevices`. 

**To Reproduce**

- Start with bluetooth device selected and active
- Switch to a different output (e.g. Earpiece or Speaker) Note: `availableAudioDevices` will still show bluetooth
- Select and activate bluetooth device again
- Select and activate different output again
RESULT: `availableAudioDevices` no longer shows bluetooth device

**Expected behavior**
The bluetooth device should not disconnect and remain listed in `availableAudioDevices`

**Android Device (please complete the following information):**
 - Device: Pixel 3XL
 - API Version: API 29

**Version**
- Version: 0.1.4

Edit: Here is the error I'm seeing:
```
Bluetooth sco job timed out
    java.util.concurrent.TimeoutException
        at com.twilio.audioswitch.bluetooth.BluetoothScoJob$BluetoothScoRunnable.run(Unknown Source:65)
        at android.os.Handler.handleCallback(Handler.java:883)
        at android.os.Handler.dispatchMessage(Handler.java:100)
        at android.os.Looper.loop(Looper.java:214)
        at android.app.ActivityThread.main(ActivityThread.java:7356)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

Edit2: It looks like there were two issues, not sure if you want to add checks for them. The first was caused by calling activate more than once. The other, was from calling start -> stop -> start again on an instance of `AudioManagerSelector`

**Comment by Alton09** [comment]
Hi @hscissors ! Thank you for reporting this issue. I was unable to reproduce this bug while following the reproduction steps on the [Twilio Video App](https://github.com/twilio/twilio-video-app-android), which uses the latest AudioSwitch release as a dependency. Do you mind sharing the source code or code snippets so that I can help troubleshoot further? Thanks!

**Comment by Alton09** [comment]
Hi @hscissors. This issue appears to be fixed as part of the latest release, 0.2.0. Can you please confirm this so we can close this issue? Thanks!

**Comment by hscissors** [comment]
Confirmed: Fixed in `0.2.0`

**Comment by Alton09** [comment]
Thanks @hscissors ! Glad to hear this issue is resolved.
